R/conclass.R

```diff
@@ -382,7 +382,7 @@ Conos <- R6::R6Class("Conos", lock_objects=F,
     },
 
     #' @description plot panel of individual embeddings per sample with joint coloring
-    plotPanel=function(clustering=NULL, groups=NULL, colors=NULL, gene=NULL, use.local.clusters=FALSE, plot.theme=NULL, use.common.embedding=FALSE, embedding.type=NULL, adj.list=NULL, ...) {
+    plotPanel=function(clustering=NULL, embedding=NULL, groups=NULL, colors=NULL, gene=NULL, use.local.clusters=FALSE, plot.theme=NULL, use.common.embedding=FALSE, embedding.type=NULL, adj.list=NULL, ...) {
```

Comment:
No need for a separate parameter. We already have `embedding.type` and can re-use it for `use.common.embedding=TRUE`.

Reply:
This makes sense---yes, sorry I missed this on first read. I'll use `embedding.type` in `plotGraph()` as well. 

For the `use.common.embedding` conditional, if `embedding.type` is `NULL`, then it will use the last created embedding. That seems reasonable.